Cabal-syntax/src/Distribution/SPDX/LicenseId.hs

```diff
@@ -33,16 +33,17 @@ import qualified Text.PrettyPrint as Disp
 -- LicenseId
 -------------------------------------------------------------------------------
 
--- | SPDX License identifiers list v3.23
+-- | SPDX License identifiers list v3.25
 data LicenseId
     = NullBSD -- ^ @0BSD@, BSD Zero Clause License
+    | 3D_Slicer_1_0 -- ^ @3D-Slicer-1.0@, 3D Slicer License v1.0, SPDX License List 3.25
```

Comment:
It's going to be a problem

Reply:
They can't. May need to wordify it.

Reply:
The parser should be amended then as well?

Reply:
Likely, although for that we may want to support both. That said, maybe it's irrelevant to working around a Haskell syntax issue in the internal representation.